test/e2e/serial/resource_accounting_test.go

```diff
@@ -0,0 +1,258 @@
+	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
+	e2ewait "github.com/openshift-kni/numaresources-operator/test/utils/objects/wait"
+	e2epadder "github.com/openshift-kni/numaresources-operator/test/utils/padder"
+	e2ereslist "github.com/openshift-kni/numaresources-operator/test/utils/resourcelist"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var _ = Describe("[serial][disruptive][scheduler] workload resource accounting", func() {
+	var fxt *e2efixture.Fixture
+	var padder *e2epadder.Padder
+	var nrtList nrtv1alpha1.NodeResourceTopologyList
+	var nrts []nrtv1alpha1.NodeResourceTopology
+
+	BeforeEach(func() {
+		var err error
+		fxt, err = e2efixture.Setup("e2e-test-resource-accounting")
+		Expect(err).ToNot(HaveOccurred(), "unable to setup test fixture")
+
+		padder, err = e2epadder.New(fxt.Client, fxt.Namespace.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = fxt.Client.List(context.TODO(), &nrtList)
+		Expect(err).ToNot(HaveOccurred())
+
+		// we're ok with any TM policy as long as the updater can handle it,
+		// we use this as proxy for "there is valid NRT data for at least X nodes
+		policies := []nrtv1alpha1.TopologyManagerPolicy{
+			nrtv1alpha1.SingleNUMANodeContainerLevel,
+			nrtv1alpha1.SingleNUMANodePodLevel,
+		}
+		nrts = e2enrt.FilterByPolicies(nrtList.Items, policies)
+		if len(nrts) < 2 {
+			Skip(fmt.Sprintf("not enough nodes with valid policy - found %d", len(nrts)))
+		}
+
+		// Note that this test, being part of "serial", expects NO OTHER POD being scheduled
+		// in between, so we consider this information current and valid when the It()s run.
+	})
+
+	AfterEach(func() {
+		err := padder.Clean()
+		Expect(err).NotTo(HaveOccurred())
+		err = e2efixture.Teardown(fxt)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	// note we hardcode the values we need here and when we pad node.
+	// This is ugly, but automatically computing the values is not straightforward
+	// and will we want to start lean and mean.
+
+	Context("cluster with at least a worker node suitable", func() {
+		var nrtTwoZoneCandidates []nrtv1alpha1.NodeResourceTopology
+		BeforeEach(func() {
+			const requiredNumaZones int = 2
+			const requiredNodeNumber int = 1
+			// TODO: we need AT LEAST 2 (so 4, 8 is fine...) but we hardcode the padding logic to keep the test simple,
+			// so we can't support ATM zones > 2. HW with zones > 2 is rare anyway, so not to big of a deal now.
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNumaZones))
+			nrtTwoZoneCandidates = e2enrt.FilterZoneCountEqual(nrts, requiredNumaZones)
+			if len(nrtTwoZoneCandidates) < requiredNodeNumber {
+				Skip(fmt.Sprintf("not enough nodes with 2 NUMA Zones: found %d", len(nrtTwoZoneCandidates)))
+			}
+		})
+
+		It("[placement][case:1] should keep the pod pending if not enough resources available, then schedule when resources are freed", func() {
```

Comment:
since we didn't have this test in the TP, I added it to track its results & this is the test ID: 49068

Reply:
I guess tier is critical. wdyt?

Reply:
I think this is `tier2` at most